src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigureBase {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line after the class declaration.